test/cppsuite/src/component/operation_tracker.cpp

```diff
@@ -60,9 +60,6 @@ operation_tracker::load()
 {
     component::load();
 
-    if (!_enabled)
-        return;
-
     /* Initiate schema tracking. */
```

Comment:
Would asserting `_enabled` be helpful?

Reply:
Yep! I added a few assertions in 90dcb9c46d8a947b04cbd0d8835a216284d09e4c, please see `test/cppsuite/src/component/component.cpp`.